docs/developing-locust.rst

```diff
@@ -36,6 +36,16 @@ Then you can run the tests by just invoking tox in the Locust project root direc
 
     tox
 
+Format code
+===========
+
+Locust follows the black formatting standard. So please adhere to it
```

Comment:
Can you make it link to https://github.com/psf/black

Reply:
Done